pkg/controller/intrusiondetection/intrusiondetection_controller.go

```diff
@@ -378,70 +386,61 @@ func (r *ReconcileIntrusionDetection) Reconcile(ctx context.Context, request rec
 		}
 	}
 
-	esClusterConfig, err := utils.GetElasticsearchClusterConfig(context.Background(), r.client)
+	// When creating the certificate manager, pass in the logger and tenant (if one exists).
+	opts := []certificatemanager.Option{
+		certificatemanager.WithLogger(reqLogger),
+		certificatemanager.WithTenant(tenant),
+	}
+	certificateManager, err := certificatemanager.Create(r.client, network, r.clusterDomain, helper.TruthNamespace(), opts...)
 	if err != nil {
-		if errors.IsNotFound(err) {
-			r.status.SetDegraded(operatorv1.ResourceNotFound, "Elasticsearch cluster configuration is not available, waiting for it to become available", err, reqLogger)
-			return reconcile.Result{}, nil
-		}
-		r.status.SetDegraded(operatorv1.ResourceReadError, "Failed to get the elasticsearch cluster configuration", err, reqLogger)
+		r.status.SetDegraded(operatorv1.ResourceCreateError, "Unable to create the Tigera CA", err, reqLogger)
 		return reconcile.Result{}, err
 	}
 
-	if isManagedCluster {
-		if esClusterConfig.ClusterName() == render.DefaultElasticsearchClusterName {
-			msg := fmt.Sprintf("%s/%s ConfigMap must contain a 'clusterName' field that is not '%s'", common.OperatorNamespace(), relasticsearch.ClusterConfigConfigMapName, render.DefaultElasticsearchClusterName)
-			err = fmt.Errorf("Elasticsearch cluster name must be non-default value in managed clusters")
-			r.status.SetDegraded(operatorv1.InvalidConfigurationError, msg, err, reqLogger)
+	// For now, these are only needed in single-tenant mode, since for multi-tenant we don't install Kibana.
+	var esgwCertificate certificatemanagement.CertificateInterface
```

Comment:
Do we still need this at the moment ?

Reply:
Nope, forgot to remove this after rebasing on your PR (we needed it before that)